core/src/main/resources/reference.conf

```diff
@@ -73,6 +73,15 @@ data-source {
   }
 }
 
+slick {
```

Comment:
Just curious where the values here came from? Is there something specific about `2000` and `5000`?

Reply:
`batchSize = 2000` is a pre-existing value set in rawls.conf: https://github.com/broadinstitute/firecloud-develop/blob/59558bd76cc869004b850e999dc9a50d5243e4c7/base-configs/rawls/rawls.conf.ctmpl#L601 … I'm just copying it down from rawls.conf to reference.conf for the sake of thoroughness.

`fetchSize = 5000` is the value that seemed to work best for me in testing - and I realize I explained that in https://github.com/broadinstitute/firecloud-develop/pull/3253 but not here, my bad!